### PR TITLE
HW_5

### PR DESCRIPTION
**Pull request by FantaBamboocha**

### CODE REVIEW

File: hw_5/task3.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+/* I had to get to know debugger while doing this task and the task with factorial */
+
+const convertRomanToInteger = (romanString) => {
+  const romanNumbers = { I: 1, V: 5, X: 10, L: 50, C: 100, D: 500, M: 1000 };
+  let integer = 0;
+
+  for (let i = romanString.length - 1; i >= 0; i--) {
+    const current = romanString[i];
+    const next = romanString[i - 1];
```
Comment (Boobaley) [inline hw_5/task3.js]: may be previous?

File: hw_5/task3.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    if (!next) {
+      integer += romanNumbers[current];
+    } else if (romanNumbers[current] > romanNumbers[next]) {
+      integer += romanNumbers[current] - romanNumbers[next];
+      i--;
+    } else {
+      integer += romanNumbers[current] + romanNumbers[next];
+      i--;
```
Comment (Boobaley) [inline hw_5/task3.js]: quite hard solution

File: hw_5/task4.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      result[color] += 1;
+    }
+
+    return result;
+  }, {});
+
+  const glovesQuantity = Object.values(glovesCollection);
+
+  return glovesQuantity.reduce((pairQuantity, quantity) => {
+    if (quantity < 2) return pairQuantity;
```
Comment (Boobaley) [inline hw_5/task4.js]: redundant line. Math.floor return 0 if 1 / 2. + 0 will not affect the result

**Review by Boobaley — CHANGES_REQUESTED** [review]

File: hw_5/task3.js (hunk 1 above)
Comment (FantaBamboocha) [reply hw_5/task3.js]: it depends on which side to look)
for the user it will be previous, but for loop "for" it will be next)

**Pull request merged**